### PR TITLE
:arrow_upper_right: fixed :bug:

### PR DESCRIPTION
**Pull request by Prathik-Jain**

### CODE REVIEW

File: LibraryManagement/Class/Admin.vb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ﻿Public Class Admin
-    Public UID As String
+    Public Uid As String
```
Comment (Shrey98) [inline LibraryManagement/Class/Admin.vb:2]: So here it is "Uid" and everywhere in the program its like "uid"

Comment (Prathik-Jain) [reply LibraryManagement/Class/Admin.vb:2]: VB is not case sensitive.

Comment (Prathik-Jain) [reply LibraryManagement/Class/Admin.vb:2]: we'll fix it in the final clean up

Comment (Shrey98) [reply LibraryManagement/Class/Admin.vb:2]: ok

**Pull request merged**